### DISCUSSION TITLE
failed to make protobuf-c 1.4.0

### DISCUSSION THREAD

**Discussion by gddhg** [protobuf-c/protobuf-c#571] [Q&A]
./configure protobuf_LIBS=/usr/local/lib/ protobuf_CFLAGS=-I/usr/local/include/google/protobuf

  GEN      protobuf-c/protobuf-c.pb.cc
  CXX      protoc-c/protoc_gen_c-c_bytes_field.o
  CXX      protoc-c/protoc_gen_c-c_enum.o
  CXX      protoc-c/protoc_gen_c-c_enum_field.o
  CXX      protoc-c/protoc_gen_c-c_extension.o
  CXX      protoc-c/protoc_gen_c-c_field.o
  CXX      protoc-c/protoc_gen_c-c_file.o
  CXX      protoc-c/protoc_gen_c-c_generator.o
  CXX      protoc-c/protoc_gen_c-c_helpers.o
  CXX      protoc-c/protoc_gen_c-c_message.o
  CXX      protoc-c/protoc_gen_c-c_message_field.o
  CXX      protoc-c/protoc_gen_c-c_primitive_field.o
  CXX      protoc-c/protoc_gen_c-c_service.o
  CXX      protoc-c/protoc_gen_c-c_string_field.o
  CXX      protobuf-c/protoc_c_protoc_gen_c-protobuf-c.pb.o
  CXX      protoc-c/protoc_gen_c-main.o
  CXXLD    protoc-c/protoc-gen-c
ld: can't map file, errno=22 file '/usr/local/lib/' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

name -a
21.4.0 Darwin Kernel Version 21.4.0: Fri Mar 18 00:47:26 PDT 2022; root:xnu-8020.101.4~15/RELEASE_ARM64_T8101 arm64

**Comment by gddhg** [comment]
CXXLD    protoc-c/protoc-gen-c
Apple clang version 13.1.6 (clang-1316.0.21.2)
Target: arm64-apple-darwin21.4.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
 "/Library/Developer/CommandLineTools/usr/bin/ld" -demangle -lto_library /Library/Developer/CommandLineTools/usr/lib/libLTO.dylib -dynamic -arch arm64 -platform_version macos 12.0.0 12.3 -syslibroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -o protoc-c/protoc-gen-c -L/usr/local/lib protoc-c/protoc_gen_c-c_bytes_field.o protoc-c/protoc_gen_c-c_enum.o protoc-c/protoc_gen_c-c_enum_field.o protoc-c/protoc_gen_c-c_extension.o protoc-c/protoc_gen_c-c_field.o protoc-c/protoc_gen_c-c_file.o protoc-c/protoc_gen_c-c_generator.o protoc-c/protoc_gen_c-c_helpers.o protoc-c/protoc_gen_c-c_message.o protoc-c/protoc_gen_c-c_message_field.o protoc-c/protoc_gen_c-c_primitive_field.o protoc-c/protoc_gen_c-c_service.o protoc-c/protoc_gen_c-c_string_field.o protobuf-c/protoc_c_protoc_gen_c-protobuf-c.pb.o protoc-c/protoc_gen_c-main.o /usr/local/lib/ -bind_at_load /usr/local/lib/libprotoc.dylib /usr/local/lib/libprotobuf.dylib -lpthread -lz -lc++ -lSystem /Library/Developer/CommandLineTools/usr/lib/clang/13.1.6/lib/darwin/libclang_rt.osx.a
ld: can't map file, errno=22 file '/usr/local/lib/' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by zilianghan** [comment]
I got error: 
protoc-c/c_message.cc:157:65: error: ‘const class google::protobuf::OneofDescriptor’ has no member named ‘file’; did you mean ‘field’?
     vars["foneofname"] = FullNameToC(oneof->full_name(), oneof->file());
platform:  Ubuntu 18.04
Any suggentions? thanks
